lib/rubocop/cop/rspec/indexed_let.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Do not set up test data using indexes (e.g., `item_1`, `item_2`).
+      #
+      # It makers reading the test harder because it's not clear what exactly
+      # is tested by this particular example.
+      #
+      # This cop is configurable using the `MaxRepeats` option.
+      #
+      # @example
```

Comment:
I think it would be easier to understand and better if we specify the change with the `MaxRepeats` setting value in the example.

``ruby
# @example `MaxRepeats: 1 (default)`
# good
: :
# bad

# @example `MaxRepeats: 2` # good : # bad
# good
good : : # bad
# bad
```